manifests/mpm.pp

```diff
@@ -66,12 +66,18 @@
         }
       }
 
+      if $mpm == 'itk' and $::operatingsystem == 'Ubuntu' and $::operatingsystemrelease == '16.04' {
+        $packagename = 'libapache2-mpm-itk'
+      } else {
+        $packagename =  "apache2-mpm-${mpm}"
+      }
+
       if versioncmp($apache_version, '2.4') < 0 or $mpm == 'itk' {
-        package { "apache2-mpm-${mpm}":
+        package { "$packagename":
```

Comment:
These two variables will produce lint errors of quoted variables so should be unquoted.